internal/pkg/diagnostics/diagnostics_test.go

```diff
@@ -61,6 +112,7 @@ func TestRedactResults(t *testing.T) {
 
 	require.Empty(t, errOut.String())
 	require.NotContains(t, outWriter.String(), "unredacted")
+	require.NotContains(t, outWriter.String(), "END OPENSSH PRIVATE KEY")
```

Comment:
nit: you could just put the key in a variable and check for all of it. This leaves us open to a bug where we only remove the last line or something.